kura/distrib/src/main/resources/generic-arm32/deb/control/postinst

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
```

Comment:
copyright header needed?

Reply:
Don't know if we need to add it, other profiles do not have it....

Reply:
Ok, will add then